packages/@sanity/components/src/previews/MediaPreview.js

```diff
@@ -25,6 +29,7 @@ export default class MediaPreview extends React.PureComponent {
     subtitle: undefined,
     description: undefined,
     media: undefined,
+    progress: undefined,
```

Comment:
No need to define a default prop as `undefined` (as that is already the default)

Reply:
Could we remove `react/require-default-props` in the linter then?